app-productivity/taskwarrior/autobuild/templates

```diff
@@ -0,0 +1,33 @@
+Template: taskwarrior/migrate_from_2
+Type: note
+Description: Upgrading from Taskwarrior 2 Needs Manual Migration
+ Taskwarrior 3 changed the way it stores and synchronises tasks, you would need
+ to export all your previous tasks before upgrading using the following command:
```

Comment:
Taskwarrior 3 introduced a new format for storing and synchronising tasks. Before upgrading Taskwarrior, you must export all your previous data using the following command: